.name_map.json

```diff
@@ -0,0 +1,6 @@
+{
+    "PeterJCLaw": ["Peter"],
```

Comment:
Nit: We should add the other meeting attendees in here too, especially given their expressed interest to contribute if only infrequently.

Reply:
This file exists to service the actions script, rather than any form of ranking/recognition. There's a balance to be had around privacy, so I opted for now to only include the people who either have actions here or who seem strongly likely to take actions soon (i.e: have signed up to be part of the team rather than observers).